app/jobs/deposit_job.rb

```diff
@@ -4,7 +4,11 @@
 class DepositJob < BaseDepositJob
   queue_as :default
 
+  # rubocop:disable Metrics/AbcSize:
   def perform(work_version)
+    Honeybadger.context({ work_version_id: work_version.id, druid: work_version.work.druid,
+                          work_id: work_version.work.id, depositor_sunet: work_version.work.depositor.sunetid })
+
```

Comment:
this is all that's needed?  we don't need to explicitly pass the context somewhere?

Reply:
I believe that is all that is needed...HB will add this context to the alert if it sends a notification (here is an example from elsewhere in the code: https://github.com/sul-dlss/happy-heron/blob/main/app/jobs/record_embargo_release_job.rb#L13)

Reply:
Yes, that is the case AFAIK too